include/Pothos/Framework/BlockRegistry.hpp

```diff
@@ -52,6 +53,8 @@ class POTHOS_API BlockRegistry
      */
     template <typename... ArgsType>
     static Proxy make(const std::string &path, ArgsType&&... args);
+
+    static bool doesBlockExist(const std::string &path);
```

Comment:
gimme a little doxygen like the other calls

Reply:
Done